app/src/main/java/ch/sdp/vibester/database/Database.kt

```diff
@@ -10,6 +10,7 @@ import com.google.firebase.ktx.Firebase
 
 object Database {
     fun get(): FirebaseDatabase {
-        return Firebase.database("https://vibester-sdp-default-rtdb.europe-west1.firebasedatabase.app")
+        val db = Firebase.database("https://vibester-sdp-default-rtdb.europe-west1.firebasedatabase.app")
+        return db
```

Comment:
do we need the extra variable declaration here?

Reply:
no that was temporary